packages/charting/src/components/StackedBarChart/StackedBarChart.styles.ts

```diff
@@ -1,51 +1,34 @@
 import { IStackedBarChartStyleProps, IStackedBarChartStyles } from './StackedBarChart.types';
 
 export const getStyles = (props: IStackedBarChartStyleProps): IStackedBarChartStyles => {
-  const { className, width, height } = props;
+  const { className, width, barHeight } = props;
 
   const chartWidth = width;
-  const chartPadding = 10;
-  const chartHeight = height;
+  const chartHeight = barHeight;
   return {
     root: [
       'ms-StackedBarChart',
       {
-        width: chartWidth + 2 * chartPadding
+        width: chartWidth
```

Comment:
I am not really sure if it's needed here, looking at the renderings that i have attached to this PR. Some of the designs might need the flexibility of managing this padding through the container that holds this component rather than an equal padding on both sides.

---
In reply to: [209397467](https://github.com/OfficeDev/office-ui-fabric-react/pull/5906#discussion_r209397467) [](ancestors = 209397467)